zcash_client_sqlite/src/wallet.rs

```diff
@@ -339,7 +461,7 @@ pub(crate) fn insert_address<P: consensus::Parameters>(
     let mut di_be = *diversifier_index.as_bytes();
     di_be.reverse();
     stmt.execute(named_params![
-        ":account": &u32::from(account),
+        ":account": account.0,
```

Comment:
Nit: rename the param here and on line 453 to "account_id" as well (I know it doesn't really matter).